{"version": "1", "message": "ERROR: Hidden Testcases server unreachable.", "problem_id": "m19_a1", "user_id": "saad181", "score": {"sample": {"0": {"execution_time": 0.7421274185180664, "passed": false}, "1": {"execution_time": 0.17273283004760742, "passed": false}, "2": {"execution_time": 0.053119659423828125, "passed": false}, "3": {"execution_time": 0.18904471397399902, "passed": false}, "4": {"execution_time": 0.16203093528747559, "passed": false}, "5": {"execution_time": 0.044800758361816406, "passed": false}, "6": {"execution_time": 0.16562342643737793, "passed": false}, "7": {"execution_time": 0.18709325790405273, "passed": false}, "8": {"execution_time": 0.06185770034790039, "passed": false}, "9": {"execution_time": 0.04508066177368164, "passed": false}, "10": {"execution_time": 0.1071770191192627, "passed": false}, "11": {"execution_time": 0.05083131790161133, "passed": false}, "12": {"execution_time": 0.04698920249938965, "passed": false}}}, "pylint_score": "0/0"}